src/msw/webview_ie.cpp

```diff
@@ -923,6 +923,11 @@ bool wxWebViewIE::RunScript(const wxString& javascript, wxString* output)
                     dispatchResult->AddRef();
                     if ( JSONAO.Invoke("stringify", DISPATCH_METHOD, varJSONStr, 1, &varResult) )
                         varResult = varJSONStr;
+                    else
+		    {
+		        wxLogWarning(_("JSON.stringify fails when trying to convert object into JSON"));
+                        return false;
```

Comment:
I believe you should add
`dispatchResult->Release();`
before
`return false;`

Also formatting seems to be somewhat broken, perhaps there are tabs instead of spaces there?